src/sagemaker/workflow/README.rst

```diff
@@ -0,0 +1,162 @@
+        'start_date': airflow.utils.dates.days_ago(2),
+        'provide_context': True
+    }
+
+    dag = DAG('tensorflow_example', default_args=default_args,
+              schedule_interval='@once')
+
+    train_op = SageMakerTrainingOperator(
+        task_id='training',
+        config=train_config,
+        wait_for_completion=True,
+        dag=dag)
+
+    transform_op = SageMakerTransformOperator(
+        task_id='transform',
+        config=trans_config,
+        wait_for_completion=True,
+        dag=dag)
+
+    transform_op.set_upstream(train_op)
+
+Using Airflow Python Operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Airflow PythonOperator <https://airflow.apache.org/howto/operator.html?#pythonoperator>`_
+is a built-in operator that can execute any Python callables. If you want to build the SageMaker workflow in a more
+flexible way,  you could write your python callables for SageMaker operations using SageMaker Python SDK. For example:
+
+.. code:: python
+
+    from sagemaker.tensorflow import TensorFlow
+
+    # callable for SageMaker training in TensorFlow
+    def train(data, **context):
+        estimator = TensorFlow(entry_point='tf_train.py',
+                               role='sagemaker-role',
+                               framework_version='1.11.0',
+                               training_steps=1000,
+                               evaluation_steps=100,
+                               train_instance_count=2,
+                               train_instance_type='ml.p2.xlarge')
+        estimator.fit(data)
+        return estimator.latest_training_job.job_name
+
+    # callable for SageMaker batch transform
+    def transform(data, **context):
+        training_job = context['ti'].xcom_pull(task_ids='training')
+        estimator = TensorFlow.attach(training_job)
+        transformer = estimator.transformer(instance_count=1, instance_type='ml.c4.xlarge')
+        transformer.transform(data, content_type='text/csv')
+
+Then you could build your workflow using PythonOperator with Python callables defined above:
+
+.. code:: python
+
+    import airflow
+    from airflow import DAG
+    from airflow.operators.python_operator import PythonOperator
+
+    default_args = {
+        'owner': 'airflow',
+        'start_date': airflow.utils.dates.days_ago(2),
+        'provide_context': True
+    }
+
+    dag = DAG('tensorflow_example', default_args=default_args,
+              schedule_interval='@once')
+
+    train_op = PythonOperator(
+        task_id='training',
+        python_callable=train,
+        op_args=[training_data_s3_uri],
+        provide_context=True,
+        dag=dag)
+
+    transform_op = PythonOperator(
+        task_id='transform',
+        python_callable=transform,
+        op_args=[transform_data_s3_uri],
+        provide_context=True,
+        dag=dag)
+
+    transform_op.set_upstream(train_op)
+
+A workflow with SageMaker training and batch transform is finished! In this way, you could customize your Python
```

Comment:
A workflow that runs a SageMaker training job and  a batch transform job is finished. You can customize your Python callables with the SageMaker Python SDK according to your needs, and build more flexible and powerful workflows.

Reply:
Updated.